File: android_app/app/src/main/java/ru/adonixis/mfc56/network/MfcStatusService.java
package ru.adonixis.mfc56.network;


import retrofit2.http.GET;
import retrofit2.http.Header;
import retrofit2.http.Headers;
import retrofit2.http.Query;
import ru.adonixis.mfc56.model.StatusResponse;
import rx.Observable;

public interface MfcStatusService {

    @Headers("Content-Type: application/json")
    @GET("api/v1/smev-client/getAppealState")
    Observable<StatusResponse> checkStatus(@Query("dealNumber") int dealNumber, @Query("pinCode") int pinCode);

}

File: android_app/app/src/main/java/ru/adonixis/mfc56/network/MfcStatusService.kt
package ru.adonixis.mfc56.network

import retrofit2.http.GET
import retrofit2.http.Headers
import retrofit2.http.Query
import ru.adonixis.mfc56.model.StatusResponse
import rx.Observable

interface MfcStatusService {

    @Headers("Content-Type: application/json")
    @GET("api/v1/smev-client/getAppealState")
    fun checkStatus(@Query("dealNumber") dealNumber: Int, @Query("pinCode") pinCode: Int): Observable<StatusResponse>

}